src/shortestpaths/desopo-pape.jl

```diff
@@ -0,0 +1,46 @@
+"""
+    struct DEposoPapeState{T, U}
+
+An [`AbstractPathState`](@ref) designed for D`Esopo-Pape shortest-path calculations.
+"""
+struct DEsopoPapeState{T <:Real, U <: Integer} <: AbstractPathState
+    parents::Vector{U}
+    dists::Vector{T}
+end
+
+function DEsopoPape_shortest_path(g::AbstractGraph, 
```

Comment:
Functions in Julia usually start with a lower case letter.
Also, considering that `dijkstra_shortests_paths` and `bellman_ford_shortest_paths` both end with an `s`, this function probably should also do that.

Reply:
Ok, cool